### DISCUSSION TITLE
Bug: Copilot removes current line after Tab

### DISCUSSION THREAD

**Discussion by leng-yue** [community/community#8012] [Copilot Conversations] [answered]
Description:  
Copilot removes current line after Tab. 

Before tab:  
![image](https://user-images.githubusercontent.com/25119060/142941215-0d1f43df-3e8f-493f-80a5-6b080dd3759f.png)
After tab:  
![image](https://user-images.githubusercontent.com/25119060/142941232-92928b72-cb47-4966-b417-d7822ad5c281.png)
This is really annoying.

Plugin Version: 1.1.0 
Ide: WebStorm 2021.2.3 
Device: MacBook Pro (16-inch, 2021), Apple M1 
System: macOS Monterey 12.0.1

**Comment by felipeuematsu** [comment]
Happens on Windows 11 too

Plugin Version: 1.1.0
Ide: CLion 2021.2.2

**Comment by yamin8000** [comment]
I have the same problem after updating the plugin to the new version (1.1.0) today.

IntelliJ IDEA 2021.2.3 (Ultimate Edition)
Build #IU-212.5457.46, built on October 12, 2021
Runtime version: 11.0.12+7-b1504.40 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0

**Comment by AdemJensen** [comment]
Me2, got same problem after upgrade today (2021-11-23, version 1.1.0)

GoLand 2021.2.4
Build #GO-212.5457.54, built on October 16, 2021

Runtime version: 11.0.12+7-b1504.40 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.0.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Registry: suggest.all.run.configurations.from.context=true, ide.completion.variant.limit=500
Non-Bundled Plugins: com.github.copilot (1.1.0), com.intellij.plugins.thrift (1.10.0), com.bytedance.sast.goland (1.0.0.68)

**Comment by aeneasr** [comment]
Can also confirm this issue happens on upgrading to 1.1.0

**Comment by jansorg** [comment, marked as answer]
@leng-yue Thank you everyone for reporting!
This is an issue with 1.1.0. You can work around it by typing the first character after the whitespace before accepting the suggestion, e.g. `M` in the screenshot above.
This will be fixed shortly with the next update.

**Comment by jansorg** [comment]
1.1.1 should fix this issue. Please let us know when you're still seeing it!
